test/jdk/sun/net/www/protocol/jar/MultiReleaseJarURLConnection.java

```diff
@@ -77,7 +77,7 @@ public void initialize() throws Exception {
         creator.buildMultiReleaseJar();
         creator.buildSignedMultiReleaseJar();
 
-        server = new SimpleHttpServer(InetAddress.getLoopbackAddress());
+        server = new SimpleHttpServer(TESTCONTEXT,System.getProperty("user.dir", "."));
```

Comment:
Please add space after comma.

Reply:
I believe the InetAddress parameter should be preserved in order to allow test to specifically bind to the loopback address instead of the wildcard (binding to the wildcard has been a source of test instability in the past).

Reply:
Thanks. Review comments are fixed in next patch